x/profile/internal/keeper/utils_functions.go

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+)
+
+// IterateProfile iterates through the profiles set and performs the provided function
+func (k Keeper) IterateProfile(ctx sdk.Context, fn func(index int64, profile types.Profile) (stop bool)) {
```

Comment:
Can we rename this `IterateProfiles`? I think it might sound more appropriate in the plural form.